client/locale/it.po

```diff
@@ -0,0 +1,237 @@
+msgstr "Scala di risoluzione"
+
+#: client/scenes/lobby_gui.cpp:368 client/scenes/lobby_gui.cpp:372
+#, c++-format
+msgid "{} - {}x{} per eye"
+msgstr "{} - {}x{} per occhio"
+
+#: client/scenes/lobby_gui.cpp:385
+msgid "Enable microphone"
+msgstr "Attiva microfono"
+
+#: client/scenes/lobby_gui.cpp:395
+msgid "Enable eye tracking"
+msgstr "Abilita il tracciamento oculare"
+
+#: client/scenes/lobby_gui.cpp:406
+msgid "Enable face tracking"
+msgstr "Abilita il tracciamento facciale"
+
+#: client/scenes/lobby_gui.cpp:416
+msgid "Enable video passthrough in lobby"
+msgstr "Abilita la visualizzazione reale in ingresso"
+
+#: client/scenes/lobby_gui.cpp:424
+msgid "Show performance metrics"
+msgstr "Mostra metriche delle prestazioni"
+
+#: client/scenes/lobby_gui.cpp:428
+msgid "Overlay can be toggled by pressing both thumbsticks"
+msgstr "È possibile alternare la visualizzazione premendo entrambi i joystick"
+
+#: client/scenes/lobby_gui.cpp:458 client/scenes/lobby_gui.cpp:466
+#: client/scenes/stream_stats.cpp:147
+msgid "CPU time"
+msgstr "Tempo CPU"
+
+#: client/scenes/lobby_gui.cpp:462
+msgid "CPU time [ms]"
+msgstr "Tempo CPU [ms]"
+
+#: client/scenes/lobby_gui.cpp:472 client/scenes/lobby_gui.cpp:480
+#: client/scenes/stream_stats.cpp:149
+msgid "GPU time"
+msgstr "Tempo GPU"
+
+#: client/scenes/lobby_gui.cpp:476
+msgid "GPU time [ms]"
+msgstr "Tempo GPU [ms]"
+
+#: client/scenes/lobby_gui.cpp:504 client/scenes/lobby_gui.cpp:773
+msgid "Licenses"
+msgstr "Licenze"
+
+#: client/scenes/lobby_gui.cpp:763
+msgid "Server list"
+msgstr "Elenco dei server"
+
+#: client/scenes/lobby_gui.cpp:766
+msgid "Settings"
+msgstr "Impostazioni"
+
+#: client/scenes/lobby_gui.cpp:770
+msgid "About"
+msgstr "Informazioni"
+
+#: client/scenes/lobby_gui.cpp:776
+msgid "Exit"
+msgstr "Esci"
+
+#: client/scenes/stream_stats.cpp:149
+msgid "Reproject"
+msgstr "Riproiezione"
+
+#: client/scenes/stream_stats.cpp:150
+msgid "Blit"
+msgstr "Copia"
+
+#: client/scenes/stream_stats.cpp:152
+msgid "Download"
+msgstr "Download"
+
+#: client/scenes/stream_stats.cpp:153
+msgid "Upload"
+msgstr "Upload"
+
+#: client/scenes/stream_stats.cpp:222
+#, c++-format
+msgid "Decoder {}"
+msgstr "Decoder {}"
+
+#: client/scenes/stream_stats.cpp:234
+msgid "Timings [ms]"
+msgstr "Durata [ms]"
+
+#: client/scenes/stream_stats.cpp:294
+msgid "Encode"
+msgstr "Codifica"
+
+#: client/scenes/stream_stats.cpp:295
+msgid "Send"
+msgstr "Invio"
+
+#: client/scenes/stream_stats.cpp:296
+msgid "Receive"
+msgstr "Ricevi"
+
+#: client/scenes/stream_stats.cpp:297
+msgid "Decode"
+msgstr "Decodifica"
+
+#: client/scenes/stream_stats.cpp:298
+msgid "Blitted"
+msgstr "Copiato"
+
+#: client/scenes/stream_stats.cpp:299
+msgid "Displayed"
+msgstr "Visualizzato"
+
+#: client/scenes/stream_stats.cpp:300
+msgid "Predicted"
+msgstr "Previsto"
+
+#: client/scenes/stream_stats.cpp:318
+#, c++-format
+msgid "Estimated motion to photons latency: {}ms"
+msgstr "Latenza stimata del movimento dei fotoni: {}ms"
```

Comment:
It is the time between user motion to optical display. photons themselves don't have a latency.

Direi "Latenza stimata da movimento a fotoni: {}ms"